COMPILING.md

```diff
@@ -1,97 +1,44 @@
-# Alba Installation
+# Alba Development Setup
 
-> Clone the alba repo.
->> - git clone https://github.com/openvstorage/alba
+### Clone the alba repo.
 
-> Install the dependencies.
->> - sudo apt-get install libprotoc-dev protobuf-compiler libjerasure-dev yasm
-
-> Install ISA-L v2.14
->> Download the package from https://01.org/intel%C2%AE-storage-acceleration-library-open-source-version/downloads .
->> ./configure
->> make
->> sudo make install
-
-> List of required opam packages (See the file _tags under ocaml directory. You don't have to install all these package because some of them will be installed automatically during arakoon installation.)
->> arakoon.1.8.8
->> kinetic-client
->> tiny_json
->> cmdliner
->> ppx_deriving_yojson
->> lwt.2.5.0
->> oUnit
->> ctypes
->> ctypes-foreign
->> snappy
->> uuidm
->> orocksdb
-
->Run make in alba home directory.
->> - make
-
-> If any package is still missing then `make` will show the error message about missing package.
->> - ocamlfind ocamldep -package kinetic-client -package nocrypto -package uuidm -package tiny_json -package snappy.
-
->> - ocamlfind: Package `kinetic-client' not found
-
->> - Command exited with code 2.
-Compilation unsuccessful after building 1 target (0 cached) in 00:00:00.
+```
+git clone https://github.com/openvstorage/alba
+```
 
->To install the packages
->> - opam install package-name
+- Install the dependencies.
+The exact, up-to-date list of what you need can be found in the dockerfiles we
+use for CI.
 
->During the installation, you may be asked to upgrade a package. It's always preferable to pin the package (don't upgrade) specified by developer.
->> - opam pin package-name version
+*    [ubuntu/debian](./docker/alba_debian/Dockerfile).
```

Comment:
Indentation / opsommingsteken might not be exactly what you had in mind here? (See rendered version of the .md file to see what I mean)
Also the '.' doesn't look like it's in its place there...